airsonic-main/src/main/java/org/airsonic/player/dao/UserDao.java

```diff
@@ -164,7 +165,7 @@ public void deleteUser(String username) {
     public void updateUser(User user) {
         String sql = "update " + getUserTable() + " set password=?, email=?, ldap_authenticated=?, bytes_streamed=?, bytes_downloaded=?, bytes_uploaded=? " +
                 "where username=?";
-        getJdbcTemplate().update(sql, encrypt(user.getPassword()), user.getEmail(), user.isLdapAuthenticated(),
```

Comment:
Why is this not needed anymore?

Reply:
Replaced by an `update()` call to the AbstractDAO parent, which does the same thing, but a few additional things:
- logging
- timing
- argument sanitization/conversion

Literally every other Dao class uses the AbstractDAO, this class was the only exception which calls the JdbcTemplate directly. It probably flew under the radar.

It now needs to be converted to be like every other Dao class because we have important timestamp conversions happening in AbstractDAO